chat.js

```diff
@@ -676,9 +676,19 @@ class CommandContext {
 		// unknown URI, allow HTTP to be safe
 		return 'http://' + uri;
 	}
-	canHTML(html) {
+	canHTML(html, whitelist) {
```

Comment:
A whitelist should be attached to the existing groupchat permissions for `canHTML`, rather than something new entirely like this.